Sources/SpeziBluetoothServices/Services/PulseOximeterService.swift

```diff
@@ -0,0 +1,38 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ByteCoding
+import Foundation
+import NIOCore
+import SpeziBluetooth
+import SpeziNumerics
+
+
+/// Bluetooth Pulse Oximeter (PLX) Service implementation.
+///
+/// This class implements the Bluetooth [Pulse Oximeter Service 1.0.1](https://www.bluetooth.com/specifications/specs/plxs-html/).
```

Comment:
```suggestion
/// This type implements the Bluetooth [Pulse Oximeter Service 1.0.1](https://www.bluetooth.com/specifications/specs/plxs-html/).
```

Reply:
done. (i also changed it for the `WeightScaleService`, since that was what i had based by documentation text on.)